src/sprites/rendered-target.js

```diff
@@ -488,10 +488,10 @@ class RenderedTarget extends Target {
      * @param {?int} index Index at which to add costume
      */
     addCostume (costumeObject, index) {
-        if (index) {
-            this.sprite.addCostumeAt(costumeObject, index);
-        } else {
+        if (typeof index === 'undefined') {
```

Comment:
Any particular reason not to check `typeof index === 'number' && !isNaN(index)`. I suspect we wouldn't want to call `addCostumeAt` on the provided index if either of these checks is false.

Reply:
I think that would be making this a bit too smart — if you provided the string `'one'` as the index, rather than `1`, it would put it at the end. I think I would rather it throw in that case.